src/extension.ts

```diff
@@ -54,6 +60,8 @@ export async function activate(context: vscode.ExtensionContext) {
         const aquarius = new Aquarius(aquariusUrl)
         const asset = await aquarius.resolve(did)
 
+        outputChannel.appendLine(`Asset details: ${JSON.stringify(asset)}`)
```

Comment:
This also saves the output in the console? not only as a pop-up.

Reply:
Yeah. In the output tab, for example: 
![image](https://github.com/user-attachments/assets/f6d0bacc-fd11-46d3-91e6-9e856d944ca9)